liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_case_management_key-001.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS dbo.nrt_case_management_key;
```

Comment:
Does the dimension have contain the UID? This framework is best suited if the UID is available for reference in the dimension.

Reply:
No, `D_CASE_MANAGEMENT` does not contain `case_management_uid` (same situation as for `NOTIFICATION` dim)

Reply:
This might end up causing the associations to be lost if we rebuild Liquibase.